src/main/resources/plugin.yml

```
@@ -45,14 +45,15 @@ permissions:
         default: false
         children:
             skript.effectcommands: true
+            skript.reloadnotify: true
             skript.admin: true
             #skript.config: true
     skript.effectcommands:
         default: false
         description: Allows to use effects as commands, e.g. '!set health to 10'
+    skript.reloadnotify:
+        default: false
+        description: Shows when others are reloading script files.
```

```suggestion
        description: Notifies when others are reloading script files.
```
I feel like this is a better verb